Omnibus.md

```diff
@@ -3119,7 +3119,7 @@ Gain the following abilities for 1 point each:
   be done once.
 * The AC for [Death Watch](#death-watch) is changed to "Spellcaster must kneel
   on one knee holding their Implement with both hands for 60 seconds before
-  being killed." The requirement for recasting the spell is also removed.
+  being killed." [The spell no longer ends when the caster is raised.]{.new}
```

Comment:
`being killed." [The spell no longer ends when the spellcaster is raised.]{.new}`